scripts/install-service.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+set -e 
+
+# Define data directory to be used for application config file.
+DATA_DIR=${DATA_DIR:-${HOME}/.poglink}
+
+# Change to script directory for predictable behaviour
+cd "$(dirname "$0")"
+
+# Check whether virtual environment is active
+if [ -z "${VIRTUAL_ENV}" ] && [ "${FORCE}" != "true"  ]; then
+    echo "Warning: Virtual environment not detected! It is highly recommended to install this program to run under a virtual environment. Please activate your virtual environment and run this script again."
+    echo "If you're certain you'd like to continue without a virtual environment, you can run again with 'FORCE=true'." 
+    exit 1
+fi
+
+# Install poglink python package
+pip install poglink
+
+# Obtain path to python executable
+if [ -z "${VIRTUAL_ENV}" ]; then
+    echo -e "\n\nVirtual environment not detected."
+    PYTHON_EXECUTABLE="$(which python3)"
+else 
+    echo -e "\n\nVirtual environment detected."
+    PYTHON_EXECUTABLE="${VIRTUAL_ENV}/bin/python3"
+fi 
+
+echo "Configuring service to run with ${PYTHON_EXECUTABLE}"
+
+# Disable service if currently running from a previous installation
+SERVICE_FILE_PATH=/etc/systemd/system/poglink.service
+sudo systemctl stop "$SERVICE_FILE_PATH" || true
+
+# Copy service file into place
+sudo install \
+    -v \
+    --backup \
+    --suffix .bak \
+    ../resources/poglink.service \
+    ${SERVICE_FILE_PATH}
```

Comment:
I like to use `install` instead of `cp` here because it allows me to set a backup option, in case you run accidentally and overwrite a service file you've modified.